main.py

```diff
@@ -1,10 +1,46 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_description in players.items():
+        race = Race.objects.get_or_create(
+            name=player_description["race"]["name"],
+            description=player_description["race"]["description"]
+        )[0]
+
+        if player_description["guild"]:
```

Comment:
The condition 'if player_description["guild"]:' assumes that the 'guild' key is always present. It would be safer to use 'player_description.get("guild")' to avoid KeyError if the key is missing.